app/controllers/api/v2/access_requests_controller.rb

```diff
@@ -25,11 +25,13 @@ def index
       end
 
       def create
-        authorize AccessRequest # todo, generalise admin auth
         access_request = AccessRequest.new(access_request_params)
-        access_request.update(requester: @current_user,
-                              request_date_utc: Time.now.utc,
-                              status: :requested)
+        authorize access_request
+
+        access_request.requester        = User.find_by(email: access_request.requester_email)
```

Comment:
This code is beautiful, incredible work friend!

Reply:
Should this be happening in a `before_create` hook? Feels funny having this in the model ... if we ever create an access request via mcb, for example, it wouldn't do the right thing.

Reply:
Yeah, this code isn't ideal in many ways, but this is the smallest change to fix the bug. I think anything else should be addressed separately.

Reply:
Yeah, let's fix the actual issue so that our support team can go back to using this first... @misaka ok with it?

Reply:
@tvararu @heidar Now that it's fixed and shipping ... can we raise a PR to fix it "properly"? Assuming this wouldn't be part of a bigger feature/story, I think we'd rather avoid "tech debt" stories if they're not clearly attached to something else.